test/helpers/filesystem.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"github.com/spf13/afero"
+)
+
+const (
+	DummyMachineID = "dummy-machine-id"
+	DummyAgentID   = "779cdd70-e9e2-58ca-b18a-bf3eb3f71244"
+
+	machineIDPath = "/etc/machine-id"
+)
+
+// MockMachineIDFile mocks the /etc/machine-id file to have a known value
+func MockMachineIDFile() afero.Fs {
```

Comment:
I find this a bit misleading as we're mocking the whole filesystem, not only the file :sweat_smile:

Reply:
How would you call it?
I find the name together with the docstring pretty meaningful, but I wrote msyelf the function, so I'm biased

Reply:
I wouldn't call it :stuck_out_tongue_winking_eye: 

However, I can think of `MockFilesystemWithMachineID`, `MockFilesystemMachineID`, which are by no means any better than `MockMachineIDFile` at the end :sweat_smile: 

So up to you.